src/otx/algo/detection/backbones/csp_darknet.py

```diff
@@ -148,7 +149,7 @@ def __init__(
         arch_ovewrite: list | None = None,
         spp_kernal_sizes: tuple[int, ...] = (5, 9, 13),
         norm_cfg: dict | None = None,
-        act_cfg: dict | None = None,
+        activation_callable: Callable[..., nn.Module] | None = partial(nn.LeakyReLU, negative_slope=0.1),
```

Comment:
It seems that Swish is used as default, but it's changed to leakyrelu now. is it ok?

Reply:
Thanks for finding, updated at 9f0d5a7.